.github/workflows/release.yml

```diff
@@ -2,12 +2,60 @@ name: Release
 
 on:
   workflow_dispatch:
+  release:
+    types: [created]
 
 jobs:
-  test:
-    runs-on: ubuntu-18.04
+  pio-release:
+    runs-on: ubuntu-20.04
+
     steps:
-      - name: Show environment v2
-        run: env | grep ^GITHUB
-      - name: Show ref v2
-        run: echo "===============> Version from $GITHUB_REF"
+    - name: Checkout code
+      uses: actions/checkout@v2
+    
+    - name: Add SSH key
+      uses: webfactory/ssh-agent@v0.9.0
+      with:
+        ssh-private-key: ${{ secrets.SKDK_OTA_PRO_PRIVKEY }}
+    
+    - name: Set env vars
+      run: |
+        echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+        echo "BUILD_TIMESTAMP=$(date -u +'%s')" >> $GITHUB_ENV
+        echo "COMMIT_HASH=$(git rev-parse HEAD)" >> $GITHUB_ENV
+
+    - name: Cache pip
```

Comment:
Maybe we should wrap all of that into a dockerfile?

Reply:
Will look into it!